.gitignore

```diff
@@ -0,0 +1 @@
+build_artefacts
```

Comment:
Yeah, me neither. We should fix it here in the feedstocks, but am ok doing that later.

Sure, let's get it out and play. Have had some other people use this successfully too. So it is already useful.